modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceQuickAddToCart.testcase

```diff
@@ -0,0 +1,677 @@
+			KeyPress(
+				locator1 = "Publications#KEYWORD_TEXTFIELD",
+				value1 = "\ENTER");
+
+			AssertElementPresent(locator1 = "TextInput#ALERT");
+		}
+	}
+
+	@description = "This is a test for COMMERCE-10390. Buyer user can quick add to cart and complete a checkout"
+	@priority = "4"
+	test CanQuickAddToCartAndCheckout {
+		property custom.properties = "jsonws.web.service.paths.excludes=${line.separator}company.security.strangers.verify=false";
+		property portal.acceptance = "false";
+
+		task ("Given an account with a buyer user is created") {
+			CommerceEntry.addAccountEntryUser(
+				accountName = "Commerce Account",
+				accountType = "Business",
+				agreeToTermsAndAnswerReminderQuery = "true",
+				createNewAccount = "true",
+				requireReset = "false",
+				userEmailAddress = "buyer@liferay.com",
+				userFirstName = "Buyer",
+				userLastName = "User",
+				userRole = "Buyer",
+				userScreenName = "buyeruser",
+				userSiteMembership = "Minium");
+		}
+
+		task ("And buyer user logs in") {
+			User.logoutPG();
+
+			CommerceLogin.miniumLogin(
+				password = "test",
+				urlAppend = "web/minium",
+				userEmailAddress = "buyer@liferay.com");
+		}
+
+		task ("And asserts there are no existing open orders") {
+			CommerceNavigator.gotoMiniumSidebarItem(menuItem = "Pending Orders");
+
+			AssertElementPresent(locator1 = "RemoteApps#EMPTY_REMOTE_TABLE_MESSAGE");
+		}
+
+		task ("And returns to the catalog") {
+			CommerceNavigator.gotoMiniumSidebarItem(menuItem = "Catalog");
+		}
+
+		task ("When the buyer opens the mini-cart") {
+			FrontStore.addInitializerProductToCart(
+				productName = "U-Joint",
+				productQuantity = "1");
+
+			// To be removed when COMMERCE-10472 is fixed
+
+			CommerceNavigator.gotoMiniCart();
+
+			Button.click(button = "Remove All Items");
+
+			// To be removed when COMMERCE-10472 is fixed
+
+			Button.click(button = "Yes");
+
+			// To be removed when COMMERCE-10472 is fixed
+
+		}
+
+		task ("And the buyer selects multiple skus on the quick add-to-cart") {
+			for (var productSku : list "MIN38805,MIN55860,MIN93015") {
+				CommerceQuickAddToCart.searchAndSelectOrderItem(orderItemSku = "${productSku}");
+			}
+		}
+
+		task ("And the buyer clicks on the search bar add to cart button") {
+			Click(locator1 = "CommerceAccelerators#QUICK_MINI_CART_ADD_TO_CART_BUTTON");
+		}
+
+		task ("Then all selected skus are added to the cart") {
+			for (var productSku : list "MIN38805,MIN55860,MIN93015") {
+				AssertElementPresent(
+					key_cartItemSku = "${productSku}",
+					locator1 = "CommerceAccelerators#MINI_CART_ITEM_SKU");
+			}
+		}
+
+		task ("And all selected skus quantity is set as 1 by default") {
+			for (var productSku : list "MIN38805,MIN55860,MIN93015") {
+				AssertTextEquals(
+					key_cartItemSku = "${productSku}",
+					locator1 = "CommerceAccelerators#MINI_CART_ITEM_QUANTITY",
+					value1 = "1");
+			}
+		}
+
+		task ("And When the buyer starts the checkout") {
```

Comment:
I'm afraid this won't work on CI. It'd better to use CommerceCheckout.initializersCheckout

Reply:
initializersCheckout doesn't work for multiple SKUs in the checkout. Maybe I can keep using CommerceCheckout.checkout and just change the click from "Submit" to "View Details" in order to avoid issues on CI